src/lib/propagation/tests/wrapper-test.js

```diff
@@ -0,0 +1,189 @@
+      },
+      {
+        id: 4,
+        organization_id: 2,
+        imported_fact_ids: [4],
+        exported_facts_count: 2,
+      },
+    ],
+    organizations: [
+      {id: 1},
+      {id: 2},
+    ],
+    facts: {
+      globalFacts: [
+        {
+          variable_name: 'Chicago',
+          children: [
+            {
+              variable_name: 'population',
+              simulation: {
+                samples: [100],
+                errors: [],
+              },
+            },
+          ],
+        }
+      ],
+      organizationFacts: [
+        {
+          variable_name: organizationReadableId({id: 1}),
+          children: [
+            {id: 1, expression: '3', imported_to_intermediate_space_ids: [1, 2]},
+            {id: 2, metric_id: 1, exported_from_id: 1},
+            {id: 3, metric_id: 3, exported_from_id: 2},
+            {id: 7, expression: '100'},
+            {id: 8, metric_id: 10, exported_from_id: 5},
+          ],
+        },
+        {
+          variable_name: organizationReadableId({id: 2}),
+          children: [
+            {id: 4, expression: '3', imported_to_intermediate_space_ids: [4]},
+            {id: 5, metric_id: 8, exported_from_id: 4},
+            {id: 6, metric_id: 9, exported_from_id: 4},
+          ],
+        },
+      ],
+    },
+    metrics: [...space1Metrics, ...space2Metrics, ...space3Metrics, ...space4Metrics, ...space5Metrics],
+    guesstimates: [ ...space1Guesstimates, ...space2Guesstimates, ...space3Guesstimates, ...space4Guesstimates, ...space5Guesstimates ],
+    simulations: [...space1Sims, ...space2Sims, ...space3Sims, ...space4Sims, ...space5Sims],
+  }
+
+  it ("should correctly extract a single space's subset", () => {
+    const testCases = [
+      {
+        description: "Passing a single metricId should yield that metric's space's subset",
+        graphFilters: { metricId: 1 }
+      },
+      {
+        description: "Passing a single spaceId should yield that space's subset",
+        graphFilters: { spaceId: 1 }
+      },
+    ]
+
+    testCases.forEach( ({graphFilters, description}) => {
+      const {subset, relevantFacts} = getSubset(state, graphFilters)
+      expect(subset, "The subset's metrics should match").to.have.property('metrics').that.deep.has.members(space1Metrics)
+      expect(subset, "The subset's guesstimates should match").to.have.property('guesstimates').that.deep.has.members(space1Guesstimates)
+      expect(subset, "The subset's simulations should match").to.have.property('simulations').that.deep.has.members(space1Sims)
+      expect(relevantFacts, 'The relevantFacts should match').to.deep.have.members([
+        {id: 1, expression: '3', imported_to_intermediate_space_ids: [1, 2]},
+        {id: 2, metric_id: 1, exported_from_id: 1, expression: `=${expressionSyntaxPad(1)}`},
+      ])
+    })
+  })
+
+  it ("should correctly extract all possibly dependent, fact exporting space's subsets from a factId", () => {
```

Comment:
rename 'fact exporting spaces' to 'intermediate spaces' or similar language in server.